test/account_test.js

```diff
@@ -0,0 +1,153 @@
+const assert = require('assert');
+
+Feature('account').tag('@desktop')
+
+// todo: run same test with loginMobile for @mobile
+
+Scenario('I can see my account data', ({ I }) => {
+  I.amOnPage('http://localhost:8080?c=test--subscription');
+  I.login();
+
+  I.click('div[aria-label="Open user menu"]');
+  I.click('Account');
+
+  I.see('Email');
+  I.see('12345@test.org');
+  I.see('Edit account');
+
+  I.see('Security');
+  I.see('Password');
+  I.see('****************');
+  I.see('Edit password');
+
+  I.see('About you');
+  I.see('First name');
+  I.see('Last name');
+  I.see('Edit information');
+
+  I.see('Terms & tracking');
+  I.wait(3);
+  I.see('I accept the Terms and Conditions of Cleeng');
+  I.see('Yes, I want to receive videodock and videodock.org updates by email');
+  I.see('Update consents');
+});
+
+Scenario('I can edit my email', ({ I }) => {
+  I.click('Edit account');
+  I.see('Confirm password');
+
+  I.click('Cancel');
+  I.dontSee('Confirm password');
+
+  I.click('Edit account');
+  I.fillField('email', '12345@test.org');
+  I.fillField('confirmationPassword', 'Ax854bZ!$');
+  I.click('Save');
+  I.wait(2);
+  I.see('Email already exists!');
+  //todo: missing 'wrong password' message in application
```

Comment:
As Cameron mentioned yesterday, I think it would be nice to extract all todos in the backlog (or a separate list)

Reply:
I've created a separate ticket for extracting https://jwplayer.atlassian.net/browse/BC-77

Reply:
Great 👍